routers/comments-router.js

```diff
@@ -0,0 +1,10 @@
+const commentsRouter = require("express").Router();
+
+commentsRouter.all("/*", (req, res) => {
```

Comment:
Hmmm. Have a think about whether you'll even need this router if you're responding to every request that comes this way with a 404.

Reply:
Haha good point! I think I'd put it in there in anticipation as I'm about to do a comments endpoint. Is it OK to leave it rather than take out and do it again later? (please :))